### PR TITLE
[ui] showing a warning during pooling

### PR DESCRIPTION
**Pull request by thomasdavis**

### CODE REVIEW

**Comment by 59023g** [comment]
From a user standpoint, seems like something is missing.. Like a color coded string message that matches the outline. And/or color coded tooltip... as a user I don't know the context. Why is this box red or yellow? cc @caseyturnerarrington

**Comment by thomasdavis** [comment]
I still gotta make an ! icon too

**Review by 59023g — APPROVED** [review]

File: ui/core/src/entities/formulae.ts [hunk 1]
```diff
@@ -1,6 +1,31 @@
 import Big from "big.js";
 import { Fraction, IFraction } from "./fraction/Fraction";
 
+export function slipAdjustment(
```
Comment (59023g) [inline ui/core/src/entities/formulae.ts:4]: What's this?

**Review by 59023g — APPROVED** [review]
Looks good.  What're the formulae changes? Prettier?

**Pull request merged**